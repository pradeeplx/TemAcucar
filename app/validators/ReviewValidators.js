export default ReviewValidators = {
  rating: {
    title: 'Avaliação',
    validate: [{
      validator: 'isNumeric',
    }]
  },
  text: {
    title: 'Descreva sua experiência',
    validate: [{
      validator: 'isLength',
      arguments: [1, 255],
    }]
  },
  errorMessage: () => 'Oops! Ocorreu um erro ao acessar nosso servidor. Por favor, tente novamente.',
}
