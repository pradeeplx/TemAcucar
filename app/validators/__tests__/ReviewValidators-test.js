import 'react-native'
import React from 'react'
import ReviewValidators from '../ReviewValidators'

test('ReviewValidators object', () => {
  expect(ReviewValidators).toMatchSnapshot()
})

describe('ReviewValidators.errorMessage cases', () => {
  it('has a default errorMessage', () => {
    expect(ReviewValidators.errorMessage({ id: 'foo' })).toMatchSnapshot()
  })
})