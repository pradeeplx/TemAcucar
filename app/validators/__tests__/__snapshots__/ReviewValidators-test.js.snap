exports[`ReviewValidators.errorMessage cases has a default errorMessage 1`] = `"Oops! Ocorreu um erro ao acessar nosso servidor. Por favor, tente novamente."`;

exports[`test ReviewValidators object 1`] = `
Object {
  "errorMessage": [Function],
  "rating": Object {
    "title": "Avaliação",
    "validate": Array [
      Object {
        "validator": "isNumeric",
      },
    ],
  },
  "text": Object {
    "title": "Descreva sua experiência",
    "validate": Array [
      Object {
        "arguments": Array [
          1,
          255,
        ],
        "validator": "isLength",
      },
    ],
  },
}
`;
